ScreenManager.c

```diff
@@ -24,7 +24,7 @@ in the source distribution for its full text.
 #include "XUtils.h"
 
 
-ScreenManager* ScreenManager_new(Header* header, const Settings* settings, const State* state, bool owner) {
+ScreenManager* ScreenManager_new(Header* header, const Settings* settings, State* state, bool owner) {
```

Comment:
What's the reason of removing this `const`?

Reply:
The ScreenManager now changes the state in the event of the pound key pressed:
```c
case '#':
         this->state->hideMeters = !this->state->hideMeters;
         ScreenManager_resize(this);
         force_redraw = true;
```

The action code only works in the MainPanel, not in settings etc..